crates/test_utils/src/lib.rs

```diff
@@ -202,6 +202,14 @@ pub fn parse_fixture(fixture: &str) -> Vec<FixtureEntry> {
     res
 }
 
+/// Same as `parse_fixture`, except it allow empty fixture
+pub fn parse_single_fixture(fixture: &str) -> Option<FixtureEntry> {
+    if !fixture.lines().any(|it| it.trim_start().starts_with("//-")) {
+        return None;
+    }
+    parse_fixture(fixture).into_iter().nth(0)
```

Comment:
We probably should check that it's the *only* fixture?

Reply:
Fixed.